.github/workflows/check-generated.yml

```diff
@@ -0,0 +1,37 @@
+# Verify that generated code is up-to-date.
+
+name: Check generated code
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - '*'
+
+permissions:
+  contents: read
+
+jobs:
+  check-error-doc:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Golang
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.20'
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+
+      - name: Check generated error docs
+        run: |
+          make error-doc-gen
+          if ! git diff --stat --exit-code ; then
+            echo ">> ERROR:"
+            echo ">>"
+            echo ">> Error documents require update (source files in x folder may have changed)."
+            echo ">> Ensure your tools are up-to-date, re-run 'make error-doc' and update this PR."
+            echo ">>"
+            exit 1
+          fi
```

Comment:
Why does this file is added in this file?

Reply:
This was added automatically when I click github ui to change base. I'll fix it.